nextflow_schema.json

```diff
@@ -518,7 +518,7 @@
                 },
                 "amp_ampcombi_cutoff": {
                     "type": "number",
-                    "default": 0.4,
+                    "default": 0.0,
```

Comment:
Not sure if to use int or float here because the ampcombi bug wouldn't allow floats. Probably just change to 0 (int) until the ampcombi bug is solved?

Reply:
(In `nextflow.config` it's written as 0)

Reply:
Yeah, maybe make an issue on ampCombi, but we can merge this in now at least (but shouldn't release pipeline until that in and we can switch it back to a float)

Reply:
Lets make ampcombi not fail, and satisfy linting. This otherwise should be fixed in ampcombi

Reply:
[AMPcombi issue](https://github.com/Darcy220606/AMPcombi/issues/93) :heavy_check_mark:

Reply:
:+1: Last two comments below to merge in and then Good to go!